src/composables/aeSdk.ts

```diff
@@ -39,26 +39,32 @@ import {
 } from '@/protocols/aeternity/config';
 import { useAccounts } from './accounts';
 import { useModals } from './modals';
+import { useNetworks } from './networks';
 
 let aeSdk: AeSdkSuperhero;
-let dryAeSdk: AeSdk;
 let aeSdkBlocked = false;
-const isAeSdkReady = ref(false);
 let aeSdkCurrentNetwork: INetwork;
+const isAeSdkReady = ref(false);
 const nodeNetworkId = ref<string>();
 const aeppInfo: Record<string, any> = {};
 
+let dryAeSdk: AeSdk;
+let dryAeSdkCurrentNodeNetworkId: string;
+
 export function useAeSdk({ store }: IDefaultComposableOptions) {
+  const { activeNetwork } = useNetworks();
   const { isLoggedIn, activeAccount } = useAccounts({ store });
   const { openModal } = useModals();
 
+  // TODO replace with the `useAeNetworkSettings` after resolving the circular dependencies
```

Comment:
https://github.com/aeternity/superhero-wallet/pull/2241
Was circular dependencies fixed by this PR?

Reply:
Yes, good finding. I'll update the PR regarding this.